### ISSUE TITLE
5.1 audio track to Stéréo without losing 5.1

### ISSUE THREAD

**Issue by Victor-root** [staxrip/staxrip#378]
I have a complete series with each episodes having 1 audio track in 5.1 and I would like to convert it to Stereo, the problem is that I don't want to replace it but add it.

So manually I have to put the audio track in the container to then re-encode the new track in Stereo.

It works, the only problem is that I have 24 episodes per season and doing them one by one is extremely long.

Is it possible to make the process automatic?

To summarize :

Each episode has a 5.1 audio track, I want to convert it to Stereo without losing 5.1. All automatically for each episode.

Thank you !

![container](https://user-images.githubusercontent.com/65600486/97113814-001ea280-16ed-11eb-8422-0c3b144b88ba.PNG)
![staxrip main](https://user-images.githubusercontent.com/65600486/97113815-001ea280-16ed-11eb-9e47-dab6d6b188a3.PNG)

**Comment by vonH** [comment]
When you open the file and it demuxes, what's the directory output (since the source appears to be 'MULTI')?
